pkg/config/user_config.go

```diff
@@ -122,6 +122,10 @@ type GuiConfig struct {
 	NerdFontsVersion string `yaml:"nerdFontsVersion" jsonschema:"enum=2,enum=3,enum="`
 	// If true (default), file icons are shown in the file views. Only relevant if NerdFontsVersion is not empty.
 	ShowFileIcons bool `yaml:"showFileIcons"`
+	// Whether to show full author names or their shortened form in the commit graph.
+	// One of 'auto' (default) | 'full' | 'short'
+	// If 'auto', initials will be shown in small windows, and full names - in larger ones.
+	CommitAuthorFormat string `yaml:"commitAuthorFormat" jsonschema:"enum=short,enum=full`
```

Comment:
I think you're missing a `"` at the end of the jsonschema tag definition.

Reply:
We should also include the `auto` enum member, as that is the default value.

Reply:
thanks for catching this, fixed